orca-core/src/main/groovy/com/netflix/spinnaker/orca/config/OrcaConfiguration.groovy

```diff
@@ -83,7 +82,7 @@ class OrcaConfiguration {
     Duration.of(3, MINUTES)
   }
 
-  @Bean Scheduler scheduler() {
+  @Bean(destroyMethod = '') Scheduler scheduler() {
```

Comment:
rxjava 1.0.15 introduces a shutdown method on Schedulers that Spring autodetects which does not play nice with closing and refreshing a context within a JVM (as is the case in unit tests)